tests/subsys/ipc/pbuf/ipc.icmsg_pbuf fails to buila at h20: 'CONFIG_DCACHE_LINE_SIZE' undeclared here

**Describe the bug**
For
- tests/subsys/ipc/pbuf/ipc.icmsg_pbuf


At:

    - nrf54h20dk@0.9.0/nrf54h20/cpuapp


**To Reproduce**
Run twister for given configuration for given platform.

**Expected behavior**
Valid console output

**Impact**
Not clear

**Logs and console output**
```
CMakeFiles/soc__nordic.dir/common/dmm.c.obj -c /jenkins_cache/test-zephyr-sanitycheck/zephyr/zephyr/zephyr/soc/nordic/common/dmm.c
In file included from ../../../../../../../../../zephyr/include/zephyr/sys/util_macro.h:34,
                 from ../../../../../../../../../zephyr/include/zephyr/sys/util.h:17,
                 from ../../../../../../../../../zephyr/include/zephyr/sys/atomic.h:24,
                 from ../../../../../../../../../zephyr/include/zephyr/kernel_includes.h:25,
                 from ../../../../../../../../../zephyr/include/zephyr/kernel.h:17,
                 from ../../../../../../../../../zephyr/include/zephyr/cache.h:16,
                 from ../../../../../../../../../zephyr/soc/nordic/common/dmm.c:7:
../../../../../../../../../zephyr/soc/nordic/common/dmm.h:38:42: error: 'CONFIG_DCACHE_LINE_SIZE' undeclared here (not in a function); did you mean 'CONFIG_TIMESLICE_SIZE'?
   38 |         (DMM_IS_REG_CACHEABLE(node_id) ? CONFIG_DCACHE_LINE_SIZE : sizeof(uint32_t))
      |                                          ^~~~~~~~~~~~~~~~~~~~~~~
../../../../../../../../../zephyr/include/zephyr/sys/util_internal.h:72:26: note: in definition of macro '__DEBRACKET'
   72 | #define __DEBRACKET(...) __VA_ARGS__
      |                          ^~~~~~~~~~~
../../../../../../../../../zephyr/include/zephyr/sys/util_internal.h:64:9: note: in expansion of macro '__GET_ARG2_DEBRACKET'
   64 |         __GET_ARG2_DEBRACKET(one_or_two_args _if_code, _else_code)
      |         ^~~~~~~~~~~~~~~~~~~~
../../../../../../../../../zephyr/include/zephyr/sys/util_internal.h:59:9: note: in expansion of macro '__COND_CODE'
   59 |         __COND_CODE(_XXXX##_flag, _if_1_code, _else_code)
      |         ^~~~~~~~~~~
../../../../../../../../../zephyr/include/zephyr/sys/util_macro.h:210:9: note: in expansion of macro 'Z_COND_CODE_1'
  210 |         Z_COND_CODE_1(_flag, _if_1_code, _else_code)
      |         ^~~~~~~~~~~~~
../../../../../../../../../zephyr/soc/nordic/common/dmm.c:14:9: note: in expansion of macro 'COND_CODE_1'
   14 |         COND_CODE_1(DT_NODE_HAS_PROP(node_id, zephyr_memory_attr), (fn(node_id)), ())
      |         ^~~~~~~~~~~
../../../../../../../../../zephyr/soc/nordic/common/dmm.c:26:22: note: in expansion of macro 'DMM_REG_ALIGN_SIZE'
   26 |          .dt_align = DMM_REG_ALIGN_SIZE(node_id),                                                  \
      |                      ^~~~~~~~~~~~~~~~~~
../../../../../../../../../zephyr/soc/nordic/common/dmm.c:14:69: note: in expansion of macro '_BUILD_MEM_REGION'
   14 |         COND_CODE_1(DT_NODE_HAS_PROP(node_id, zephyr_memory_attr), (fn(node_id)), ())
      |                                                                     ^~

```

**Environment (please complete the following information):**
 - OS: Ubuntu 24.04.0 LTS
 - Toolchain Zephyr SDK 1.0.0
 - Commit SHA or Version used: `a3e41914e33e9bf0169307c8b3696bb98bc7f926`

--- a/soc/nordic/common/dmm.h
+++ b/soc/nordic/common/dmm.h
@@ -23,7 +23,7 @@ extern "C" {
 
 /** @cond INTERNAL_HIDDEN */
 
-#ifdef CONFIG_DCACHE
+#if defined(CONFIG_DCACHE) && defined(CONFIG_CACHE_MANAGEMENT)
 
 /* Determine if memory region is cacheable. */
 #define DMM_IS_REG_CACHEABLE(node_id)					     \
@@ -42,7 +42,7 @@ extern "C" {
 #define DMM_IS_REG_CACHEABLE(node_id) 0
 #define DMM_REG_ALIGN_SIZE(node_id) (sizeof(uint32_t))
 
-#endif /* CONFIG_DCACHE */
+#endif /* CONFIG_DCACHE && CONFIG_CACHE_MANAGEMENT */
 
 /* Determine required alignment of the data buffers in memory region
  * associated with specified device node.

